Tasks/DotNetCoreCLIV2/task.loc.json

```diff
@@ -120,7 +120,7 @@
       "type": "string",
       "label": "ms-resource:loc.input.label.arguments",
       "defaultValue": "",
-      "visibleRule": "command = build || command = publish || command = run || command = test || command = custom",
+      "visibleRule": "command = build || command = publish || command = run || command = test || command = test || command = custom",
```

Comment:
Duplicate 'test', shouldn't one be 'pack' ?

Reply:
You're right, thanks.